cli/src/main/java/org/openjdk/skara/cli/GitJCheck.java

```diff
@@ -325,10 +325,12 @@ else if (confFile || confWorkingTree || workingTree) {
         if (staged) {
             ranges.clear();
             ranges.add(STAGED_REV);
+            System.out.println("When jcheck is running on staged, the following checks are not available: DuplicateIssuesCheck, IssuesCheck, IssuesTitleCheck, MergeMessageCheck, MessageCheck, ProblemListsCheck, ReviewersCheck.");
         }
         if (workingTree) {
             ranges.clear();
             ranges.add(WORKING_TREE_REV);
+            System.out.println("When jcheck is running on working-tree, the following checks are not available: DuplicateIssuesCheck, IssuesCheck, IssuesTitleCheck, MergeMessageCheck, MessageCheck, ProblemListsCheck, ReviewersCheck.");
```

Comment:
Instead of listing the checks that aren't available, I think we should list the checks that are and base that list on the actual list in JCheck.java. A manually created message like this will likely get outdated in the future.

Reply:
Right, will fix it. Thanks!